### PR TITLE
fix(MaOption): fix indicator shrinking

### PR DESCRIPTION
**Pull request by sergigraciaj**
When we have a long text with more than one line, the indicator shrinks:
image

### CODE REVIEW

**Review by alex-castelo — APPROVED** [review]
I think all it's OK!

File: src/components/MaOption/MaOption.css [hunk 1]
```diff
@@ -81,6 +81,7 @@
     height: 1rem;
     position: relative;
     transition: border 0.4s, background-color 0.33s;
+    flex-shrink: 0;
```
Comment (edufarre) [inline src/components/MaOption/MaOption.css:84]: Shall we add the required prefix for some of the browsers that require it?
https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink#browser_compatibility

**Review by edufarre — COMMENTED** [review]
Quick question, shouldn't the `indicator` class be `display: flex` in order to use `flex-shrink`?

**Comment by alex-castelo** [comment]
> Quick question, shouldn't the `indicator` class be `display: flex` in order to use `flex-shrink`?

I think indicator it's son of `ma-.radio` selector which does have the `display:flex` property. It should work if I'm not wrong. Maybe I'm missing something.

**Review by edufarre — APPROVED** [review]

**Review by Samitier — APPROVED** [review]

**Pull request merged**